lib/trivial.nix

```diff
@@ -1,17 +1,44 @@
 rec {
 
-  # Identity function.
+  /* The identity function
+     For when you need a function that does “nothing”.
+
+     Type: id :: a -> a
```

Comment:
Is that a new idiom? I don't remember seeing other functions being described by their types.

If it's a new introduction, can I suggest always putting it right next to the function definition?